gapic/api/artman_bigtable.yaml

```diff
@@ -6,46 +6,65 @@ common:
   proto_deps:
     - google-common-protos
   import_proto_path:
-    - ${REPOROOT}/googleapis
+    - ${GOOGLEAPIS}
   src_proto_path:
-    - ${REPOROOT}/googleapis/google/bigtable/v2
+    - ${GOOGLEAPIS}/google/bigtable/v2
   service_yaml:
-    - ${REPOROOT}/googleapis/google/bigtable/bigtable.yaml
+    - ${GOOGLEAPIS}/google/bigtable/bigtable.yaml
   output_dir: ${REPOROOT}/artman/output
+  git_repos:
+    staging:
+      location: git@github.com:googleapis/api-client-staging.git
   gapic_api_yaml:
-    - ${REPOROOT}/googleapis/google/bigtable/v2/bigtable_gapic.yaml
+    - ${GOOGLEAPIS}/google/bigtable/v2/bigtable_gapic.yaml
 csharp:
   gapic_code_dir: ${REPOROOT}/artman/output/gcloud-csharp-cloud-bigtable
 go:
   gapic_code_dir: ${REPOROOT}/gapi-cloud-bigtable-go
-  git_repo:
-    location: git@github.com:GoogleCloudPlatform/google-cloud-go.git
-    gapic_subpath: bigtable/apiv2
+  git_repos:
+    go:
+      location: git@github.com:GoogleCloudPlatform/google-cloud-go.git
+      gapic_subpath: bigtable/apiv2
+    staging:
+      gapic_subpath: google-cloud-bigtable-v2/cloud.google.com/go/bigtable/apiv2
 java:
-  gapic_code_dir: ${REPOROOT}/google-cloud-java/google-cloud-bigtable
-  git_repo:
-    gapic_subpath: generated/java/google-cloud-bigtable-v2
+  gapic_code_dir: ${REPOROOT}/artman/output/java/google-cloud-bigtable
+  git_repos:
+    java:
+      gapic_component: src/main/java/com/google/cloud/bigtable/spi/v2
+      gapic_subpath: google-cloud-pubsub/src/main/java/com/google/cloud/bigtable/spi/v2
```

Comment:
* The value that I see here in `gapic_component` is generated inside toolkit for java; this doesn't need to be set at all.
* `gapic_subpath` should just be `google-cloud-pubsub`